test/integration/generated/bodyString/src/operations/stringOperations.ts

```diff
@@ -0,0 +1,434 @@
+/*
+ * Copyright (c) Microsoft Corporation.
```

Comment:
Looks like new files are being added for the renamed operation groups, you might need to delete the leftover files with the old name